integration-test/trino-it/docker-compose.yaml

```diff
@@ -15,11 +15,13 @@ services:
     entrypoint:  /bin/bash /tmp/hive/init.sh
     volumes:
       - ./init/hive:/tmp/hive
+      - ../build/trino-ci-container-log/hive:/tmp/root
+      - ../build/trino-ci-container-log/hdfs:/usr/local/hadoop/logs
     healthcheck:
       test: ["CMD", "/tmp/check-status.sh"]
       interval: 10s
       timeout: 60s
-      retries: 5
+      retries: 10
```

Comment:
Is this check duplicated with the `checkContainerStatus` in `HiveContainer.java`?

Reply:
No, it's use by docker comopse